src/lib.rs

```diff
@@ -670,6 +670,7 @@ macro_rules! steps {
     (
         $worldtype:path => { $( $items:tt )* }
     ) => {
+        #[doc(hidden)]
```

Comment:
Any reason we couldn't use `#[allow(missing_docs)]` in this position?

Reply:
They both suppress the warning, it's just a question of whether you want it to show up in the docs or not.

Reply:
The path of least astonishment for me would be to simply suppress the warning. In general, the tests should be in a `#[cfg(test)]` situation, so in general documentation shouldn't be generated for it in the first place, but in those cases, the user is likely to being it intentionally and may wish to handle this differently.